sound/soc/sof/topology.c

```diff
@@ -2234,8 +2234,7 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 	}
 
 	ipc_size = sizeof(struct sof_ipc_comp_process) +
-		le32_to_cpu(private->size) +
-		ipc_data_size;
+		ipc_data_size + sizeof(swidget->comp_ext);
```

Comment:
Is private->size truly no longer relevant?

Reply:
I actually can't understand why we put this private->size here previously, @juimonen @singalsu do you recall it?

Reply:
Hi @paulstelian97 I think the private->size was added by mistake, I just test with my removal the process components works fine, including the sof-ctl to get/set tlv data for it.

I think the reason why no any issue reported for this is that though we are passing bigger ipc_size, but we only fill data to sof_ipc_comp_process and .data[], and the fw takes only "sizeof(struct sof_ipc_comp_process) + ipc_data_size", so this error here is not fatal and nobody have found it. :)